From daa2bad1e233433d264b221904dcfd932bfdc033 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Tue, 27 Aug 2024 13:58:56 +0530
Subject: [PATCH] base_aaos: Make 6.6 as default kernel

Tests done:
- Android boot in GVT-d config
- adb reboot
- Wi-Fi on/off/connect/disconnect in static config

Tracked-On: OAM-123636
Change-Id: I4041560f797b3ac48c719c7ddfc948af6b9f6e0c
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 mixins.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mixins.spec b/mixins.spec
index d13832e..5482585 100755
--- a/mixins.spec
+++ b/mixins.spec
@@ -7,7 +7,7 @@ mixinsrel: false
 product.mk: device.mk
 
 [groups]
-kernel: gmin64(useprebuilt=false,src_path=kernel/linux-intel-lts2022, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/linux-intel-lts2022, more_modules=true, lts2021_chromium_src_path=kernel/lts2021-chromium, lts2021_chromium_cfg_path=config-lts/lts2021-chromium, linux_intel_lts2021_src_path=kernel/linux-intel-lts2021, linux_intel_lts2021_cfg_path=config-lts/linux-intel-lts2021, linux_intel_lts2022_src_path=kernel/linux-intel-lts2022, linux_intel_lts2022_cfg_path=config-lts/linux-intel-lts2022,branch=android14-6.1,kmi_generation=0,i915_ag_mods_version=true)
+kernel: gmin64(useprebuilt=false,src_path=kernel/linux-intel-lts2023, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/linux-intel-lts2023, more_modules=true, lts2021_chromium_src_path=kernel/lts2021-chromium, lts2021_chromium_cfg_path=config-lts/lts2021-chromium, linux_intel_lts2021_src_path=kernel/linux-intel-lts2021, linux_intel_lts2021_cfg_path=config-lts/linux-intel-lts2021, linux_intel_lts2022_src_path=kernel/linux-intel-lts2022, linux_intel_lts2022_cfg_path=config-lts/linux-intel-lts2022,branch=android14-6.1,kmi_generation=0,i915_ag_mods_version=true)
 disk-bus: auto
 boot-arch: project-celadon(fw_sbl=true,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=base_aaos,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true)
 sepolicy: enforcing
-- 
2.45.2

